tests/Unit/Adapter/ToolsTest.php

```diff
@@ -49,7 +49,11 @@ public function provideTestCasesForBcAdd(): iterable
      */
     public function testRefreshCaCertFile(): void
     {
-        @unlink(_PS_CACHE_CA_CERT_FILE_);
+        if (file_exists(dirname(_PS_CACHE_CA_CERT_FILE_))) {
+            @unlink(_PS_CACHE_CA_CERT_FILE_);
+        } else {
+            mkdir(dirname(_PS_CACHE_CA_CERT_FILE_));
```

Comment:
But maybe you wanted to know if directory exists to create it if necessary ?

I don't understand your intention with the test `file_exists(dirname(_PS_CACHE_CA_CERT_FILE_))` I think ^^'
Do you test if file exists to remove it or do you test if directory exists to create it if necessary ?